src/morphology/english/getVerbForms.js

```diff
@@ -164,9 +164,9 @@ const endsWithS = function( word ) {
  * @returns {boolean} True if the word ends with "ing".
  */
 const endsWithIng = function( word ) {
-	const wordLength = word.length;
-	// Consider only words of five letters or more to be ing forms (otherwise, words like "ping" are being treated as verb forms).
-	if ( wordLength > 4 ) {
+	const vowelCount = ( word.match( /([aeiouy])/g ) || [] ).length;
```

Comment:
If the regex is defined as a constant at the top of the file, it has to be compiled only once.
Right now it is defined and compiled every time this function is called.
This could save some time when calling this function repeatedly. (Or I am just being paranoid).

Reply:
indeed!